docs/ref/contrib/admin/index.txt

```diff
@@ -437,16 +437,22 @@ subclass::
 
     * ``description``
         A string of optional extra text to be displayed at the top of each
-        fieldset, under the heading of the fieldset. This string is not
-        rendered for :class:`~django.contrib.admin.TabularInline` due to its
-        layout.
+        fieldset, under the heading of the fieldset.
 
         Note that this value is *not* HTML-escaped when it's displayed in
         the admin interface. This lets you include HTML if you so desire.
         Alternatively you can use plain text and
         :func:`django.utils.html.escape` to escape any HTML special
         characters.
 
+    .. admonition:: :class:`~django.contrib.admin.TabularInline` does not support fieldsets
+
+        Using ``fieldsets`` in conjunction with ``TabularInline`` has no effect,
```

Comment:
Thank you for reviewing. Do you think mentioning only the things that can be done would be better? I am trying to avoid being too explicit to prevent the need for frequent updates if things change. Would that be better? Here is something I am looking to change it to.


```
.. admonition:: :class:`~django.contrib.admin.TabularInline` has limited support for fieldsets

    Using ``fieldsets`` with ``TabularInline`` has limited functionality. When used, it only allows you to:

    - Specify which fields will be available
    - Define the order in which the fields will appear within the ``TabularInline`` layout

```

Reply:
I think that's a clever idea ⭐
```
    .. admonition:: ``TabularInline`` has limited support for ``fieldsets``

        Using ``fieldsets`` with :class:`~django.contrib.admin.TabularInline`
        has limited functionality. You can specify which fields will be
        displayed, and in which order, within the ``TabularInline`` layout by
        defining ``fields`` in the ``field_options`` dictionary.

        All other features are not supported. This includes the use of ``name``
        to define a title for a group of fields.
```
I think that you cannot group fields with a name is maybe worth mentioning as this is perhaps the "core" use of defining a fieldset in the first place 🤔 what do you think?

Reply:
> I think that's a clever idea ⭐
> 
> ```
>     .. admonition:: ``TabularInline`` has limited support for ``fieldsets``
> 
>         Using ``fieldsets`` with :class:`~django.contrib.admin.TabularInline`
>         has limited functionality. You can specify which fields will be
>         displayed, and in which order, within the ``TabularInline`` layout by
>         defining ``fields`` in the ``field_options`` dictionary.
> 
>         All other features are not supported. This includes the use of ``name``
>         to define a title for a group of fields.
> ```
> 
> I think that you cannot group fields with a name is maybe worth mentioning as this is perhaps the "core" use of defining a fieldset in the first place 🤔 what do you think?

Yes it looks good, will update and add the test